planb/update_cluster.py

```diff
@@ -44,22 +44,25 @@ def text_timestamp():
 
 
 def select_keys(d: dict, keys: list):
-    return {k: v for k,v in d.items() if k in keys}
+    return {k: v for k, v in d.items() if k in keys}
 
 
 def create_tags(ec2: object, resource_id: str, tags: dict):
     # TODO: make sure this is retried on throttling/ratelimit error
     ec2.create_tags(Resources=[resource_id],
-                    Tags=[{'Key': k, 'Value': v} for k, v in tags.items()])
+                    Tags=[{'Key': k, 'Value': v}
+                          for k, v in tags.items()])
 
 
 def tags_as_dict(tags: list) -> dict:
     return {t['Key']: t['Value'] for t in tags}
 
 
 def update_tags(ec2: object, resource_id: str, tags: dict):
-    create_tags(ec2, resource_id,
-                dict(tags, **{'planb:operation:last-modified': text_timestamp()}))
+    create_tags(
+        ec2, resource_id,
+        dict(tags, **{'planb:operation:last-modified': text_timestamp()})
+    )
```

Comment:
I think that in this case readability would suffer greatly if we would try to put the closing brackets on the same line:
```
    user_data_changes = {
        'volumes': {
            'ebs': {
                '/dev/xvdf': "{}-{}".format(options['cluster_name'],
                                            saved_instance['PrivateIpAddress'])}}}
```
This is plain unreadable, IMO.
On the other hand, when it comes to function calls it would be exactly 1 level of nesting most of the time, so I don't see a benefit of putting the closing paren on a separate line.  For me this only explodes the vertical space.